src/main/java/net/schmizz/sshj/DefaultConfig.java

```diff
@@ -96,6 +98,19 @@ private String readVersionFromProperties() {
         }
     }
 
+    private boolean readBouncycastleRuntimeRegistrationProperty() {
+        try {
+            Properties properties = new Properties();
+            properties.load(DefaultConfig.class.getClassLoader().getResourceAsStream("sshj.properties"));
+            String property = properties.getProperty("sshj.bouncycastle.runtime.registration");
```

Comment:
This approach requires reading a file from the class path, it seems like it would be better to avoid the lookup and make this a simple member variable of the `DefaultConfig` class.

Reply:
Its already possible to set the variable programmatically by using the DefaultConfig.

Reply:
Thanks for the clarification, that makes sense.

Reply:
Thanks for the feedback. I actually misunderstood. I have taken out the property so that you can only set it via the ```DefaultConfig```. Any other feedback you have?

Reply:
any updates? :)